services/QuillLMS/engines/evidence/lib/evidence/evidence/check.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module Evidence
+  module Check
+    ALL_CHECKS = [
+      Prefilter,
+      RegexSentence,
+      Opinion,
+      Plagiarism,
+      AutoML,
+      RegexPostTopic,
+      Grammar,
+      Spelling,
+      RegexTypo
+    ]
+
+    FALLBACK_RESPONSE = {
+      feedback: "Thank you for your response.",
+      feedback_type: "autoML",
+      optimal: true,
+    }
```

Comment:
I like the fallback feedback here.  Might make sense in a later pass to add a new `feedback_type` for fallback to differentiate it from legitimate autoML feedback, but that can wait.

Reply:
Yeah, this is a direct copy from the Go endpoint, but agreed long-term we might want to differentiate.